### ISSUE TITLE
Reduce postinstall noise

### ISSUE THREAD

**Issue by GriffinSauce** [rsms/estrella#34]
Hi, installing our monorepo dependencies this banner gets printed:

```
> estrella@1.3.0 postinstall /Users/xyz/projects/abc/node_modules/estrella
> node dist/npm-postinstall.js


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Welcome to fun development with Estrella & esbuild!
Create a build.js file to get started. Here's a simple one:

#!/usr/bin/env node
const { build } = require("estrella")
const pkg = require("./package.json")
build({
  entry: "src/main.ts",
  outfile: pkg.main,
})

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

I appreciate the friendly greeting, but it's quite a huge amount of output for one dependency among many. If every depency did this I'd run out of buffer in my terminal quite quickly.

Besides that, if I just installed it I'm already looking at the docs, with proper context. If I or a team member installed it earlier and am (re)installing all dependencies this not likely to be aligned to my current needs.

Can I suggest reducing it to two lines maximum, linking to the documentation instead?

```
Welcome to fun development with Estrella & esbuild!
Create a build.js file to get started: https://github.com/rsms/estrella#example-use
```

It bears mentioning: Estrella rocks, thank you! 🙏

**Comment by rsms** [comment]
Maybe it would be best to simply remove that message :-)

**Comment by GriffinSauce** [comment]
That would be great too, didn't want to rain on your parade too much haha, cheers

**Comment by rsms** [comment]
Shipped in [v1.4.1](https://github.com/rsms/estrella/releases/tag/v1.4.1)
